migrator/migrations/m_170_to_m_171_create_policy_categories_and_edges/migration_test.go

```diff
@@ -50,24 +51,29 @@ func (s *categoriesMigrationTestSuite) TearDownTest() {
 func (s *categoriesMigrationTestSuite) TestMigration() {
 	ctx := sac.WithAllAccess(context.Background())
 	testPolicy := fixtures.GetPolicy()
-	testPolicy.Categories = []string{"Test Category"}
-
+	testPolicy.Categories = []string{"Test Category", "test category", "", "Anomalous Activity"}
 	require.NoError(s.T(), s.policyStore.Upsert(ctx, testPolicy))
 
+	testPolicy2 := fixtures.GetPolicy()
+	testPolicy2.Id = uuid.NewV4().String()
+	testPolicy2.Name = "testpolicy2"
+	testPolicy2.Categories = []string{"Test Category", "test category", "", "net new category", "Net new category"}
+	s.Require().NoError(s.policyStore.Upsert(ctx, testPolicy2))
+
 	dbs := &types.Databases{
 		PostgresDB: s.db.DB,
 		GormDB:     s.db.GetGormDB(),
 	}
 
 	s.NoError(migration.Run(dbs))
 
-	q := search.NewQueryBuilder().AddExactMatches(search.PolicyCategoryName, testPolicy.GetCategories()[0]).ProtoQuery()
+	q := search.NewQueryBuilder().AddExactMatches(search.PolicyCategoryName, testPolicy.Categories[0]).ProtoQuery()
```

Comment:
[nit] Can you also add a test that validates testPolicy2 after migration only has 2 categories? `Test Category` and `Net New Category`? By querying policy store directly not category store

Reply:
unfortunately, the interface doesn't have get by query so i cannot